flink-python/pyflink/common/dependency_manager.py

```diff
@@ -0,0 +1,121 @@
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+import json
+import os
+import uuid
+
+__all__ = ['DependencyManager']
+
+
+class DependencyManager(object):
+    """
+    Container class of dependency-related parameters. It collects all the dependency parameters
+    and transmit them to JVM before executing job.
+    """
+
+    PYTHON_FILE_PREFIX = "python_file"
+    PYTHON_REQUIREMENTS_PREFIX = "python_requirements_list_file"
+    PYTHON_REQUIREMENTS_DIR_PREFIX = "python_requirements_dir"
```

Comment:
Change to python_requirements_cached_dir? To make it consistent with `def set_python_requirements(self, requirements_list_file, requirements_cached_dir=None):`

And maybe change python_requirements_list_file to python_requirements_file?